_gsocproposals/2020/proposal_MCnetRivet.md

```diff
@@ -0,0 +1,48 @@
+---
+title: MCnet/Rivet - Speed and accuracy in the LHC's MC analysis tool
+layout: gsoc_proposal
+project: MCnet
+year: 2020
+organization:
+  - UGlasgow
+  - CERN
+---
+
+# Description
+
+Rivet is a software package for performing data analysis on simulated particle collision events like those in the Large Hadron Collider. Analyses of real data have to deal with the effects of the giant and complex particle detectors that surround the beam interaction points, but statistical techniques mean that much faster methods can be employed to test theory models against that data. Rivet is the LHC's principle tool for such lightweight model testing, using events simulated using Monte Carlo (MC) methods. There are currently nearly 900 analysis routines registered on the Rivet platform.
```

Comment:
This sentence doesn't make much sense to me. "Analyses of real data have to deal with the effects of the giant and complex particle detectors that surround the beam interaction points, but statistical techniques mean that much faster methods can be employed to test theory models against that data"

Much faster than which methods? What are the implications of having "to deal with the effects of the giant and complex particle detectors that surround the beam interaction points"? "statistical techniques mean that much faster methods can be employed"?